scripts/npm-safe-publish.sh

```diff
@@ -42,6 +44,12 @@ if [[ "$NPM_SHOW" != "$NPM_LS" ]]; then
   echo "Versions are not the same, (Remote = $NPM_SHOW; Local = $NPM_LS). Checking for publish eligibility."
 
   if [[ "$NPM_REGISTRY" = "https://registry.npmjs.org/" ]]; then
+    # make sure we're able to track this deployment
+    if [[ -z "$DEVCYCLE_PROD_SLEUTH_API_TOKEN" ]]; then
+      echo "Sleuth.io deployment tracking token not found. Aborting."
+      exit 1
```

Comment:
We shouldn't fail a deployment because of a failure to get the key this late - if it's truly required we should fail immediately